harness/determined/cli/workspace.py

```diff
@@ -25,6 +25,22 @@
 workspace_arg: Arg = Arg("--workspace-name", type=str, help="workspace name")
 
 
+def get_workspace_id_from_args(args: Namespace) -> Optional[int]:
```

Comment:
Did you change anything here? This looks the same as in #5716, not sure why it's showing up in the diff

Reply:
yeah I'm not sure either. 🤔

Reply:
post merge it shows up correctly 

![image](https://user-images.githubusercontent.com/12127420/212417095-b7b7fff3-dde6-43e7-bbc5-5cbaf8fdf704.png)